### PR TITLE
Carol Medina hw01 submission

### PR DESCRIPTION
**Pull request by medcar8879**
I installed GitHub Desktop which allowed me to easily clone the repo onto my device. I was then able to open the README file in RStudio and edit the file. Originally, the Git tab did not show up. After asking a lab mate, I realized I had to open a project for the Git tab to show up. After committing my changes, I tried to pull before pushing.  I got an error message telling me I was denied access/permission to the repo. After looking the error up online, I realized that I was trying to add my changes to the original class repo and not my personal, cloned copy on my Github. I think when I cloned the repo, I copied the link from the cfss github and not my own. I restarted the process, copied/cloned the link from my github, and was then able to commit/push/pull.

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* I appreciate how you "debug" your problem by asking your friend, going online, etc. This is definitely a valuable skill in coding.
* You complete all the requirements for the HW. Great job :)
* In the future, you don't need to push the .Rproj file onto your github.

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X     |              |            |
| **Coding strategy**         |           |      X        |            |
| **Presentation**            |          |       X       |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* I appreciate how you "debug" your problem by asking your friend, going online, etc. This is definitely a valuable skill in coding.
* You complete all the requirements for the HW. Great job :)
* In the future, you don't need to push the .Rproj file onto your github.

**Review by ericgao29 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Information is well organized.
* Maybe next time you can try to insert links into some words by using: `[some words](links)`

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# hw01
\ No newline at end of file
+# hw01
+## Biography
+My name is Carol Medina. I am a psychology graduate student at the University of Chicago (https://psychology.uchicago.edu/directory/carol-medina). I live in Hyde Park with my cat, **Luna**. Some of my favorite things to do include:
```
Comment (wanitchayap) [inline README.md:3]: This link should look like this:
[link](https://psychology.uchicago.edu/directory/carol-medina)
and this is how you do it:
`[link](https://psychology.uchicago.edu/directory/carol-medina)`

**Pull request closed without merging**